scripts/tools/InitializeQueue.php

```diff
@@ -66,9 +67,17 @@ class InitializeQueue extends InstallAction
 {
     use ServiceLocatorAwareTrait;
 
-    const BROKER_MEMORY = 'memory';
-    const BROKER_RDS = 'rds';
-    const BROKER_SQS = 'sqs';
+    public const BROKER_MEMORY = 'memory';
+    public const BROKER_RDS = 'rds';
+    public const BROKER_NEW_SQL = 'newsql';
+    public const BROKER_SQS = 'sqs';
+
+    public const AVAILABLE_BROKERS = [
+        self::BROKER_MEMORY,
+        self::BROKER_RDS,
+        self::BROKER_NEW_SQL,
+        self::BROKER_SQS,
+    ];
 
     private $wantedBroker;
```

Comment:
Can you please add phpdocs to the class attributes?

Reply:
done